code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -521,6 +526,9 @@ Normal range for a defender's bullet resist should be something around 30-50. ~N
 		to_chat(world, "<span class='debuginfo'>Initial armor is: <b>[armor]</b></span>")
 		#endif
 		var/penetration = P.ammo.penetration > 0 || armor > 0 ? P.ammo.penetration : 0
+		if(src == P.shot_from.sniper_target(src))
```

Comment:
This should be handled in the gun, not in defense.

Reply:
How would you make it do more damage from the gun-side specifically without having unintended effects... like say doing way more damage than you should to a wall (which is why I set it up like this)?

Reply:
Adding a damage modifier to the gun while there is a targeted mob.

Reply:
That doesn't address the problem of doing more damage to walls, and other non-mob structures you can hit than you should.

Reply:
Can you even target those? If so, why?